pkg/genericapiserver/genericapiserver.go

```diff
@@ -627,6 +630,25 @@ func (s *GenericAPIServer) Run(options *ServerRunOptions) {
 	glog.Fatal(http.ListenAndServe())
 }
 
+// If the file represented by path exists and
+// accessable, return true otherwise return false.
+func canReadFile(path string) bool {
+	if _, err := os.OpenFile(path, os.O_RDWR, 0666); err != nil {
```

Comment:
Consider just using os.Open, I think it has defaults you want. (And you probably don't want O_RDWR anyway?)